### DISCUSSION TITLE
Rewrites between two Next apps

### DISCUSSION THREAD

**Discussion by lkarcol** [vercel/next.js#25652] [Help]
Hi, 
is possible rewrite from one Next app to second next app ?
 I have 2 apps, **dashboard.domain.com/p/:slug** (SSR)  and **domain.com** . 
I created rewrite rule in domain.com app. 

`
module.exports = {
  async rewrites() {
    return [
      {
        source: "/:slug",
        destination: process.env.NEXT_DASHBOARD_URL + "p/:slug",
      },
    ];
  },
  future: {
    webpack5: true,
  },
};
`
 rewrite its work, if is opened domain.com/something i can see page from dashboard.domain
 but this page is not interactive because i get  404 errors for some chunk files. 

I think is beacuse next is trying download chunks from domain.com/_next/static  but i need files from dashboard.domain.com/_next/static  . 
Is there some workaround ?

**Comment by ghost** [comment]
facing same issue,
when the main app loads the external redirects, it doesn't load the static folder of the redirected app (for ex: _next/static/).
It is trying to access static assets from the main app's static folder which doesn't contain the required assets for the redirected app.

**Comment by wiiiimm** [comment]
Can you please try to take a look at this example: https://github.com/vercel/next.js/tree/canary/examples/with-zones ?
Based on this example, I have created another example at https://40049-nextjs-with-zones.vercel-support.app using the repo at https://github.com/vercel-support/40049-nextjs-with-zones.

**Comment by pz129** [comment]
Has there been any movement on this?

**Comment by Melvynx** [comment]
yes agree, impossible to rewrite with 2 nextjs app for now.

We get this : 


```
GET /_next/static/css/aaa3cb5b7bf61faa.css?dpl=dpl_7vr3BXKDS8Z9WeKPLmkaqKKQj4QU 404 in 1449ms
 GET /_next/static/media/569ce4b8f30dc480-s.p.woff2 404 in 1468ms
 GET /_next/static/media/93f479601ee12b01-s.p.woff2 404 in 1461ms
 GET /_next/static/chunks/20-4a4244e426078c89.js?dpl=dpl_7vr3BXKDS8Z9WeKPLmkaqKKQj4QU 404 in 146ms
 GET /_next/static/chunks/webpack-c64a44388923c105.js?dpl=dpl_7vr3BXKDS8Z9WeKPLmkaqKKQj4QU 404 in 148ms
 GET /_next/static/chunks/main-app-30549448294cf267.js?dpl=dpl_7vr3BXKDS8Z9WeKPLmkaqKKQj4QU 404 in 147ms
 GET /_next/static/chunks/40bf5d38-66a8fbc9220bd8fc.js?dpl=dpl_7vr3BXKDS8Z9WeKPLmkaqKKQj4QU 404 in 149ms
 GET /_next/static/chunks/bbe1418d-3d35464077904e52.js?dpl=dpl_7vr3BXKDS8Z9WeKPLmkaqKKQj4QU 404 in 143ms
 GET /_next/static/chunks/947-797d284d87d5a351.js?dpl=dpl_7vr3BXKDS8Z9WeKPLmkaqKKQj4QU 404 in 142ms
 GET /_next/static/chunks/app/layout-e1b16980ec5812ac.js?dpl=dpl_7vr3BXKDS8Z9WeKPLmkaqKKQj4QU 404 in 129ms
 GET /_next/static/chunks/252-c04182b586fb5200.js?dpl=dpl_7vr3BXKDS8Z9WeKPLmkaqKKQj4QU 404 in 131ms
 GET /_next/static/chunks/827-25c0d78bd4d4f905.js?dpl=dpl_7vr3BXKDS8Z9WeKPLmkaqKKQj4QU 404 in 131ms
 GET /_next/static/chunks/899-e390df4a3d3da1bb.js?dpl=dpl_7vr3BXKDS8Z9WeKPLmkaqKKQj4QU 404 in 134ms
 GET /_next/static/chunks/940c3aa4-60f56fc99b9b32e8.js?dpl=dpl_7vr3BXKDS8Z9WeKPLmkaqKKQj4QU 404 in 133ms
 GET /_next/static/chunks/186-8f00c7e36aae0b19.js?dpl=dpl_7vr3BXKDS8Z9WeKPLmkaqKKQj4QU 404 in 133ms
 GET /_next/static/chunks/116-898032dc71d45e7c.js?dpl=dpl_7vr3BXKDS8Z9WeKPLmkaqKKQj4QU 404 in 110ms
 GET /_next/static/chunks/app/nextfullstack/page-001b5e6947b93995.js?dpl=dpl_7vr3BXKDS8Z9WeKPLmkaqKKQj4QU 404 in 109ms
 GET /_next/static/chunks/531-327cec332875c864.js?dpl=dpl_7vr3BXKDS8Z9WeKPLmkaqKKQj4QU 404 in 109ms
 GET /_next/static/chunks/158-bc646b9910504c5f.js?dpl=dpl_7vr3BXKDS8Z9WeKPLmkaqKKQj4QU 404 in 111ms
 GET /_next/static/chunks/719-691f5db9adb65950.js?dpl=dpl_7vr3BXKDS8Z9WeKPLmkaqKKQj4QU 404 in 114ms
```

Basically the app try to fetch ressources to the current domain instead of rewrite domain.
